<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="11" time="3.166" timestamp="2025-03-13T07:40:23.739515-07:00" hostname="pop"><testcase classname="tests.test_environment" name="test_scipy_import" time="0.372" /><testcase classname="tests.test_environment" name="test_matplotlib_import" time="0.021" /><testcase classname="tests.test_environment" name="test_torch_import" time="0.002" /><testcase classname="tests.test_environment" name="test_output_directories" time="0.004" /><testcase classname="tests.test_environment" name="test_system_resources" time="0.002" /><testcase classname="tests.test_environment" name="test_torch_memory_management" time="0.939" /><testcase classname="tests.test_environment" name="test_python_version" time="0.003" /><testcase classname="tests.test_environment" name="test_required_packages" time="0.003" /><testcase classname="tests.test_environment" name="test_environment_variables" time="0.002"><failure message="NameError: name 'Path' is not defined">fork/tests/test_environment.py:160: in test_environment_variables
    base_dir = str(Path(__file__).parent.parent.parent)
E   NameError: name 'Path' is not defined
        python_path = ''</failure></testcase><testcase classname="tests.test_environment" name="test_numpy_import" time="0.001" /><testcase classname="tests.test_environment" name="test_pandas_import" time="0.212" /></testsuite></testsuites>